package.json

```diff
@@ -109,68 +110,68 @@
     "react-chartjs-2": "^3.0.4",
     "react-circular-progressbar": "^2.0.4",
     "react-dom": "^17.0.2",
-    "react-draggable": "^4.4.3",
-    "react-intl": "^5.20.7",
-    "react-router-dom": "^5.2.0",
+    "react-draggable": "^4.4.4",
```

Comment:
It can be removed (with another PR - see https://github.com/thorchain/asgardex-electron/issues/1752)